src/main/resources/application.properties

```diff
@@ -23,3 +23,6 @@ logging.level.org.springframework=INFO
 
 # Maximum time static resources should be cached
 spring.web.resources.cache.cachecontrol.max-age=12h
+
+#Used for testing the app with another port. In this case I used port 8095 for testing
```

Comment:
```suggestion
# Used for testing the app with another port. In this case, I used port 8095 for testing
```
And I don't think is needed.

Reply:
> And I don't think is needed.
@RenzoBot  I agree with @andrzejsydor since to most of us, the port issue is obvious but I don't mind it if the formatting is corrected as @andrzejsydor suggested :)